src/snowflake/snowpark/modin/plugin/_internal/indexing_utils.py

```diff
@@ -911,21 +912,25 @@ def get_valid_col_positions_from_col_labels(
         indexer = []
         for col_loc_item in col_loc:
             # for each locator, we perform prefix matching for multiindex
-            try:
-                if is_scalar(col_loc_item):
-                    col_loc_item = (col_loc_item,)
-                item_indexer = columns.get_locs(col_loc_item)
-            except KeyError:
-                # Note Snowpark pandas will ignore out-of-bound labels
-                item_indexer = []
+            if is_scalar(col_loc_item):
+                col_loc_item = (col_loc_item,)
+            # May throw KeyError if any labels are not found in columns
```

Comment:
You prob. can move the comment into the docstrings, there's for google style e.g. https://google.github.io/styleguide/pyguide.html#doc-function-raises `Raises:` as comment section. That also displays quickly to the user what's going on!

Reply:
I think this comment should stay since the `KeyError` can be thrown in the pandas function call, but we can also add the `Raises` to the docstring since it's thrown elsewhere too.